chall01/rjaakonm.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python3
+
+import sys
+
+morse = [".-",
+	"-...",
+	"-.-.",
+	"-..",
+	".",
+	"..-.",
+	"--.",
+	"....",
+	"..",
+	".---",
+	"-.-",
+	".-..",
+	"--",
+	"-.",
+	"---",
+	".--.",
+	"--.-",
+	".-.",
+	"...",
+	"-",
+	"..-",
+	"...-",
+	".--",
+	"-..-",
+	"-.--",
+	"--.."]
+
+def main():
+	if (len(sys.argv) != 2) or (len(sys.argv[1]) == 0):
+		print("usage: ./rjaakonm.py <a-zA-Z string>")
+		return
+	str = ""
+	sys.argv[1] = sys.argv[1].lower()
+	for char in sys.argv[1]:
+		if char.isalpha():
+			str += morse[ord(char) - 97]
```

Comment:
This solution works but if I can give you an advice, you should check python `dict`. If you're familiar with the concept of a "hash table" it's basically the same. It woud have allowed you to associate the letters with the corresponding morse code. For example `morse['a'] -> ".-"`.